lib.sh

```diff
@@ -969,11 +995,9 @@ any_key() {
 }
 
 lowest_compatible_nc() {
-if [ -z "$NC_UPDATE" ]
+if [ -z "$CURRENTVERSION" ]
```

Comment:
Why this change?

Reply:
because I want to get rid of the NC_UPDATE variable sometime in the future.
Currentversion is only set, if the new functions was called.

Reply:
Btw: now that i've changed all variables, NC_UPDATE isn't anymore called regularly